schemas/people-schema.json

```diff
@@ -83,8 +104,21 @@
       "type": "array",
       "title": "Community roles",
       "description": "Only people who have role on pull request community",
-      "examples": [["Admin"]]
+      "examples": [
+        [
+          "Admin"
+        ]
+      ]
+    },
+    "customImage": {
+      "$id": "#/properties/customImage",
+      "type": "boolean",
```

Comment:
type sting 
and need to provide the name of the image

Reply:
Then, add `"customImage": true` to your profile JSON that you created in the previous step.
 
This is what written in task.